pkgs/by-name/sd/SDL2_image/package.nix

```diff
@@ -36,16 +37,20 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
   ];
 
-  buildInputs = [
-    SDL2
-    giflib
-    libXpm
-    libjpeg
-    libpng
-    libtiff
-    libwebp
-    zlib
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ Foundation ];
+  buildInputs =
+    [
+      SDL2
+      giflib
+      libXpm
```

Comment:
While we are here: `libXpm` and `giflib` are being flagged by my `nix-check-deps` tool - are they actually required?